.github/workflows/rust.yml

```diff
@@ -13,9 +13,13 @@ jobs:
     outputs:
       crates: ${{ steps.get_matrix.outputs.crates }}
       deps: ${{ steps.get_matrix.outputs.deps }}
+      nightly_version: ${{ steps.read_toolchain.outputs.nightly_version }}
     steps:
       - name: Checkout Crate
         uses: actions/checkout@v4
+      - name: Read nightly version
+        id: read_toolchain
+        run: echo "::set-output name=nightly_version::$(cat .github/nightly-version)"
```

Comment:
You need to append this to `$GITHUB_OUTPUT` (with `>>`). Since not doing so is deprecated due to security problems.

Reply:
Oh, good to know, thanks!